__tests__/app.test.js

```diff
@@ -50,6 +50,60 @@ describe("/api/topics", () => {
 });
 
 describe("/api/articles", () => {
+  describe("GET", () => {
+    test("200 + returns all articles with correct properties and values", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const articles = body.articles;
+          expect(articles.length).toBe(13);
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                article_id: expect.any(Number),
+                title: expect.any(String),
+                topic: expect.any(String),
+                author: expect.any(String),
+                comment_count: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+              })
+            );
+          });
+        });
+    });
+    test("200 + returns all articles sorted by created_at", () => {
```

Comment:
We could delete this test block and add expect(articles).toBeSortedBy("created_at", { descending: true }); to the previous one, it should achieve the same result 🙂